htdocs/accountancy/bookkeeping/card.php

```diff
@@ -53,18 +53,22 @@
 $accountingaccount = new AccountingAccount($db);
 $accountingjournal = new AccountingJournal($db);
 
-$accountingaccount_number = GETPOST('accountingaccount_number', 'alphanohtml');
+// If subledger account is present, force general account with centralist account
+if (GETPOST("subledger_account", "alphanohtml") > 0) {
+    $subledger_account = GETPOST("subledger_account", "alphanohtml");
+
+    $subledgeraccount = new AccountingAccount($db);
+    $accountingaccount_number = $subledgeraccount->searchAccountSubledgerInfo($subledger_account);
```

Comment:
Why forcing the general account as soon as a subledger account is searched ?
If the user want to have only subledger account with a general account defined, he can also enter the filter in general account.
If there is some data with a general account and we want to see all of them, even if general account differs, so we can fix data anomalies, how can you do this then ?